packages/gluestick/src/renderer/render.js

```diff
@@ -60,9 +60,7 @@ module.exports = function render(
   );
   const isEmail = !!currentRoute.email;
   const routerContext = <RouterContext {...renderProps} />;
-  const rootWrappers = entriesPlugins
-    .filter(plugin => plugin.meta.wrapper)
-    .map(({ plugin }) => plugin);
+  const rootWrappers = entriesPlugins.filter(plugin => plugin.ref.meta.wrapper);
```

Comment:
I think I'm finally starting to understand some of these plugin lines after this PR.